.packit.yaml

```
@@ -9,35 +9,27 @@ synced_files:
 upstream_package_name: packitos
 downstream_package_name: packit
 upstream_project_url: https://github.com/packit-service/packit
-current_version_command: ["python3", "setup.py", "--version"]
-create_tarball_command: ["python3", "setup.py", "sdist", "--dist-dir", "."]
-create_pr: false
+
+actions:
+  create-archive:
+    - ["python3", "setup.py", "--version"]
```

shouldn't it be like this?
```suggestion
    - "python3 setup.py --version"
```
I'd say that you declared a list of lists.

---

I know, fix to packit is coming soon..;) (We probably want to support this syntax.)

---

But I'll use the string so we can merge this sooner.

---

okay, my point is that I think that your input is incorrect and we just need a better error message:
```
In [15]: x = """ 
    ...: asd: 
    ...: - ["a", "b", "c"] 
    ...: """                               

In [16]: x                                 
Out[16]: '\nasd:\n- ["a", "b", "c"]\n'

In [17]: yaml.safe_load(x)                 
Out[17]: {'asd': [['a', 'b', 'c']]}
```

---

https://github.com/packit-service/packit/pull/663

---

> okay, my point is that I think that your input is incorrect

No, that's definitely what I wanted. I want to (try to) use one command with the shlex parsed syntax (list) of the command. (So `[["the", "command"]]` (one command) instead of `["the", "command"]`, what will run two commands.)

> and we just need a better error message:

:+1: